reference/robot.md

```diff
@@ -378,6 +599,78 @@ double wb_robot_battery_sensor_get_value();
 int wb_robot_get_battery_sampling_period(WbDeviceTag tag);
 ```
 
+%tab-end
+
+%tab "C++"
+
+```cpp
+#include <webots/Robot.hpp>
+
+namespace webots {
+  class Robot : public Device {
+    virtual void batterySensorEnable(int sampling_period);
+    virtual void batterySensorDisable();
+    int batterySensorGetSamplingPeriod();
+    double batterySensorGetValue() const;
+    // ...
+  }
+}
+```
+
+%tab-end
+
+%tab "Python"
+
+```python
+from controller import Robot
+
+class Robot (Device):
+    def batterySensorEnable(self, sampling_period):
+    def batterySensorDisable(self):
+    def batterySensorGetSamplingPeriod(self):
+    def batterySensorGetValue(self):
+    # ...
+```
+
+%tab-end
+
+%tab "Java"
+
+```java
+import com.cyberbotics.webots.controller.Robot;
+
+public class Robot extends Device {
+  public void batterySensorEnable(int sampling_period);
+  public void batterySensorDisable();
+  public int batterySensorGetSamplingPeriod();
+  public double batterySensorGetValue();
+  // ...
+}
+```
+
+%tab-end
+
+%tab "MATLAB"
+
+```matlab
+wb_robot_battery_sensor_enable(sampling_period)
+wb_robot_battery_sensor_disable()
+period = wb_robot_battery_sensor_get_sampling_period()
+value = wb_robot_battery_sensor_get_value()
+```
+
+%tab-end
+
+%tab "ROS"
+
+| name | service/topic | data type | data type definition |
+| --- | --- | --- | --- |
+| `/battery_sensor/value` | `topic` | webots_ros::Float64Stamped | [`Header`](http://docs.ros.org/api/std_msgs/html/msg/Header.html) `header`<br/>`float64 data` |
```

Comment:
Here we miss the `/battery_sensor/enable` and  `/battery_sensor/get_sampling_period` services documentation